### DISCUSSION TITLE
娱乐至死了吗？波兹曼错了吗？ | /cn/2019/06/postman-amusing/

### DISCUSSION THREAD

**Discussion by yihui** [rbind/yihui#1319] [Comments]
https://yihui.org/cn/2019/06/postman-amusing/

[a comment by an automated account is not shown]

**Reply by yihui** [reply]
我实在没感觉世界在明显变好，只能说表面上看世界对一（小）部分人来说变好了，这一小部分人可以更麻溜地压榨大部分人了。我估计在这个问题上我们俩现在无法达成一致，不过意见不一致也很正常，大家都是盲人摸象，我的目的并不是要说服你。

至于孩子的问题，我的样本量太小了，我也不想推广我的结论。可能你的亲戚朋友的娃都碰巧很优秀、很自律，反正我看见的多数都很颓废、很让大人伤脑筋。估计等你有了娃以后，会更深刻地感受到互联网和手机的负面作用，以及发挥它们的正面作用是多么困难（是的，你可以学更多知识，但你首先得过得了垃圾广告视频这一关；就算你过得了这一关，如何让你的娃也能过这一关，起码这对我来说是个超级难题）。

在任何环境中都会产生最优秀的人，但我们不能光盯着最优秀的人看，他们只是七十亿人中的极少一部分。我关心的是普通人。

> ——原帖发布于 2019-08-16 16:04:26

[a comment by an automated account is not shown]

**Reply by yihui** [reply]
我不是要号召人读书，读不读书不是关键。不读书的人也可以劳动，也可以把下一代培养得比他们更好。我的父母都是农民，也没什么学历，也不读书，但问题是他们都是付出了大半辈子的极度辛苦劳动。如果那些小哥哥小姐姐都有正经职业，那我对他们没什么好说的。我关心的点在于现在的娱乐的成瘾性越来越强，可能会让人越来越没心思做他们的正经职业，我担心的是那些小哥哥小姐姐的观众。

当然这一切都建立在一个假设条件上，那就是未来我们仍然需要劳动来养活自己。我没法说这个假设条件在未来还一定成立。也许物质条件会富足到饭来张口、衣来伸手，躺在家里什么都不用管就可以活得很舒服。如果真那样，那我这所有的担心都是多余的。

> ——原帖发布于 2019-08-16 15:49:06

**Reply by yihui** [reply]
最近了解到我之前想象的这种社会有个专门的名字，叫“后稀缺时代”：<https://en.wikipedia.org/wiki/Post-scarcity_economy>

> ——原帖发布于 2019-10-03 18:13:52

[a comment by an automated account is not shown]

**Reply by zhangjiuliang** [reply]
收费了吗，看不了了

[a comment by an automated account is not shown]

**Comment by thomas126147** [comment]
對，未來的人們可能會越來越笨，但有何不可呢？在以往狩獵時代的人們，人人都是生存專家，他們詳細了解自己的領地，感官完美適應的領地，他們了解記住了大量動植物的訊息，也可以製作複雜的工具，這需要多年的精心訓練跟出色的運動技巧，僅僅只是過著日常生活，就能夠擁有媲美運動員的生理素質，無論智力、肉體，哪一項拿到現代人面前都是屌虐，這也是理所當然的，畢竟史前時代，在沒有社會保護下的人類，要生存就必須這麼拼命。

綜觀來看，人類的文明越是進步，人類的個體能力都更加的疲弱。

農耕社會的出現讓人們不必擁有優秀的體能也能夠擁有糧食；社會的分工，讓我們不必學會所有技能，也能擁有優渥的生活；工業革命更是改變了所有人，讓產量大幅增加，也擁有了經濟的雛形，當然，人們也變得更加的弱小。

但是別忘了，以物種的角度來說，我們的時間還太短了，不足以產生進化，又或者說是「退化」，只是因為個體不必要太過優秀也能生存下來，所以弱小的人們才會越來越多。

這是人類滅亡的前兆嗎？我哪知道，現在繁華的生活，在人類的歷史之中，其實是個特例。現代的人類生理上算起，僅僅只是延續了7500代；500代以前，農業文明開始了；20代以前，我們學會了做科學研究；互聯網的產生，也僅僅只是一代之前罷了。如今的現在，人類文明處在一個前所未有的加速，無論如何都很難準確的預測人類的未來，但比起去悲觀的看待未來，還不如盡可能的去改變這個世界，讓你所相信的理念去影響未來千千萬萬的人們吧

**Reply by yihui** [reply]
现在回过头来看，我觉得我基本上同意你的观点。谢谢分享！

**Comment by zhangjiuliang** [comment]
看不懂啊。。。。想看但是总感觉没把握住

**Reply by zhangjiuliang** [reply]
看懂了
